com/epam/sandisk/main/pages/MainPage.js

```diff
@@ -1,28 +1,33 @@
-let wait = require('../util/Wait');
-let actions = require('../util/Actions');
+const Wait = require('../util/Wait');
+let wait = new Wait();
+const Actions = require('../util/Actions');
+let actions = new Actions();
+const format = require('string-format');
 
 class MainPage {
     constructor() {
-        this.ssdRef = element(by.xpath("//div[@id='navbar']//a[@href='/home/ssd.html']"));
+        this.ssdRef = element(by.css("div#navbar a[href='/home/ssd.html']"));
         this.usbRadioButton = element(by.xpath("//label//*[contains(text(),'USB')]"));
-        this.mobileStorageImg = element(by.xpath("//*[@alt='Mobile Storage']"));
+        this.navigationBarElementXpath = "//div[@class='subnav fixed']//*[contains(text(),'{}')]";
     }
 
     open() {
-        return browser.get('https://www.sandisk.com');
+        return browser.get(browser.baseUrl);
     }
 
     clickUsbRadioButton() {
-        return Promise.resolve(actions.moveToElement(this.mobileStorageImg))
-            .then(() => wait.waitElementToBeClickable(this.usbRadioButton, 5000))
-            .then(() => this.usbRadioButton.click());
+        return actions.moveAndClick(this.usbRadioButton);
     }
 
     navigationBarContainsTag(tag) {
-        let searchElement = element(by.xpath(`//div[@class='subnav fixed']//*[contains(text(),'${tag}')]`));
-        return Promise.resolve(wait.waitElementToBeVisible(searchElement), 5000)
-            .then(() => searchElement.getText())
-            .then((text) => text === tag);
+        let searchElement = element(by.xpath(format(this.navigationBarElementXpath, tag)));
+        return wait.waitElementToBeVisible(searchElement, 5000)
```

Comment:
It will be better to include into `waitElementToBeVisible` function 5000, because it looks like as "magic number"